e2e/nocomponents.sbom

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Do we need to push `nocomponetns.sbom`, `missingmeta.sbom` and `docker-archive.sbom` files?

Reply:
Not sure we need to upload the sbom for docker-archive.sbom and missing-meta. 

We need probably need a separate task to add e2e tests which do analyze -> scan to make sure that we don't produce an SBOM that we can't analyze as a separate call.

Reply:
https://github.com/openclarity/kubeclarity/issues/507

Reply:
> Not sure we need to upload the sbom for docker-archive.sbom and missing-meta.

So please remove them from the PR

Reply:
my morning brain hadn't noticed that I'd uploaded all of them by mistake, I've added them all the to gitignore now to make sure it doesn't happen again.